### PR TITLE
docs: confluence document migration (product pt.2)

### PR DESCRIPTION
**Pull request by curlyreeve**

### CODE REVIEW

File: Product/Archive/Core-Processor/CADProc/Channel-Aggregation-And-Decisioning.md
Comment (Justus-at-Tazama) [inline Product/Archive/Core-Processor/CADProc/Channel-Aggregation-And-Decisioning.md:1]: This is not in scope for the migration, as per the spreadsheet.

File: Product/Archive/Core-Processor/Data-Preparation/Entity-Resolution-Specification-for-Implementation.md
Comment (Justus-at-Tazama) [inline Product/Archive/Core-Processor/Data-Preparation/Entity-Resolution-Specification-for-Implementation.md:1]: This should be in `docs/Knowledge-Articles/Entity-Resolution` as per the spreadsheet

File: Product/Channel-Router-And-Setup-Processor-CRSP.md
Comment (Justus-at-Tazama) [inline Product/Channel-Router-And-Setup-Processor-CRSP.md:1]: This should have been in the CRSP readme (complete) or not migrated as per the spreadsheet. This has already been migrated. This file is also duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files.

File: Product/Configuration-Management/Complete-Example-Of-A-Network-Map.md
Comment (Justus-at-Tazama) [inline Product/Configuration-Management/Complete-Example-Of-A-Network-Map.md:1]: This has already been migrated.

File: Product/Configuration-Management/Complete-Example-Of-A-Rule-Processor-Configuration.md
Comment (Justus-at-Tazama) [inline Product/Configuration-Management/Complete-Example-Of-A-Rule-Processor-Configuration.md:1]: This has already been migrated.

File: Product/Configuration-Management/Complete-Example-Of-A-Typology-Processor-Configuration.md
Comment (Justus-at-Tazama) [inline Product/Configuration-Management/Complete-Example-Of-A-Typology-Processor-Configuration.md:1]: This has already been migrated.

File: Product/Configuration-Management/Configuration-Management.md
Comment (Justus-at-Tazama) [inline Product/Configuration-Management/Configuration-Management.md:1]: This has already been migrated.

File: Product/Creating-and-Maintaining-Processors
Comment (Justus-at-Tazama) [inline Product/Creating-and-Maintaining-Processors:1]: This folder is required for:
04-Tazama-Rules-Life-Cycle
00-Introduction
01-Config-Driven-Design
02-Create-New-Rule-Processor
03-Create-New-Typology
04-Create-New-Channel
05-Create-New-Transaction

File: Product/TADPROC_Response_v0.14_20211209.json
Comment (Justus-at-Tazama) [inline Product/TADPROC_Response_v0.14_20211209.json:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/Tazama-Rules-Life-Cycle.md
Comment (Justus-at-Tazama) [inline Product/Tazama-Rules-Life-Cycle.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

This file should be in the `docs/Product/Creating-and-Maintaining-Processors` folder.

File: Product/Transaction-Aggregation-And-Decisioning-Processor-Tadproc.md
Comment (Justus-at-Tazama) [inline Product/Transaction-Aggregation-And-Decisioning-Processor-Tadproc.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/configuration-management.md
Comment (Justus-at-Tazama) [inline Product/configuration-management.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files as well as in this PR.

It has also already been migrated.

File: Product/processor-results-propagation.md
Comment (Justus-at-Tazama) [inline Product/processor-results-propagation.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/rule-processor-overview-rule-config.md
Comment (Justus-at-Tazama) [inline Product/rule-processor-overview-rule-config.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/rule-processor-overview.md
Comment (Justus-at-Tazama) [inline Product/rule-processor-overview.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/transaction-aggregation-and-decisioning-processor.md
Comment (Justus-at-Tazama) [inline Product/transaction-aggregation-and-decisioning-processor.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/transaction-monitoring-service-api.md
Comment (Justus-at-Tazama) [inline Product/transaction-monitoring-service-api.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

File: Product/typology-processing.md
Comment (Justus-at-Tazama) [inline Product/typology-processing.md:1]: This file is duplicated in PR 41 - https://github.com/frmscoe/docs/pull/41/files

**Review by Justus-at-Tazama — CHANGES_REQUESTED** [review]
Hi Laurence,

Please see and resolve review comments below. The first files in this PR turned out fine. About halfway through some of the files were going into the wrong folders and there is quite a bit of overlap between this PR and PR 41 - https://github.com/frmscoe/docs/pull/41/files

Please reconcile the files and locations against the indicated locations in the spreadsheet.

Please also note that the specific files:

- [configuration-management.md](https://github.com/frmscoe/docs/blob/dev/Product/configuration-management.md)
- [complete-example-of-a-network-map.md](https://github.com/frmscoe/docs/blob/dev/Product/complete-example-of-a-network-map.md)
- [complete-example-of-a-rule-processor-configuration.md](https://github.com/frmscoe/docs/blob/dev/Product/complete-example-of-a-rule-processor-configuration.md)
- [complete-example-of-a-typology-processor-configuration.md](https://github.com/frmscoe/docs/blob/dev/Product/complete-example-of-a-typology-processor-configuration.md)

Have all been migrated since the spreadsheet was created and last updated.

**Review by RobReeveLexTego — DISMISSED** [review]
second block for archive.
Documents are now correctly named and in the correct location

File: Product/Creating-and-Maintaining-Processors
Comment (RobReeveLexTego) [reply Product/Creating-and-Maintaining-Processors:1]: will be added back in subsequent PR

**Review by RobReeveLexTego — APPROVED** [review]
renaming Tazama Platform to Tazama System

**Review by RobReeveLexTego — DISMISSED** [review]
Changes to address conflicts

**Review by Justus-at-Tazama — APPROVED** [review]
Resolved merge conflicts caused by https://github.com/frmscoe/docs/pull/55/files

**Pull request merged**